chapters/operatorsandexpressions.tex

```diff
@@ -531,10 +531,10 @@ \subsection{Numeric Functions and Conversion Functions}\label{numeric-functions-
   \lstinline!(if leftJustified then "-" else "") + String(minimumLength) + "d"!
 \end{itemize}
 
-Form of the \lstinline!format! string:
-According to ANSI-C the format string specifies one conversion specifier (excluding the leading \%), shall not contain length modifiers, and shall not use `\lstinline!*!' for width and/or precision.
-For all numeric values the format specifiers `\lstinline!f!', `\lstinline!e!', `\lstinline!E!', `\lstinline!g!', `\lstinline!G!' are allowed.
-For integral values it is also allowed to use the `\lstinline!d!', `\lstinline!i!', `\lstinline!o!', `\lstinline!x!', `\lstinline!X!', `\lstinline!u!', and `\lstinline!c!' format specifiers (for non-integral values a tool may round, truncate or use a different format if the integer conversion characters are used).
+The ANSI-C style \lstinline!format! string (which cannot be combined with any of the other named arguments) consists of a single conversion specifier without the leading \%.
+It shall not contain a length field specifier, and shall not use `\lstinline!*!' for width and/or precision.
```

Comment:
```suggestion
It shall not contain a length modifier, and shall not use `\lstinline!*!' for width and/or precision.
```
K&R calls it length modifier, and as far as I can tell so does the C-standard (at least n1570 has a similar text).
A typical length modifier is `l` as in `ld` when writing a long int and not an int.